packages/manager/src/features/ObjectStorage/AccessKeyLanding/AccessKeyTable/HostNameTableCell.tsx

```diff
@@ -31,14 +32,14 @@ export const HostNameTableCell = ({
   if (!regionsLookup || !regionsData || !regions || regions.length === 0) {
     return <TableCell>None</TableCell>;
   }
+  const label = regionsLookup[storageKeyData.regions[0].id]?.label;
+  const s3Endpoint = storageKeyData?.regions[0]?.s3_endpoint;
 
   return (
     <TableCell>
-      {`${regionsLookup[storageKeyData.regions[0].id].label}: ${
-        storageKeyData?.regions[0]?.s3_endpoint
-      } `}
+      {`${label}: ${s3Endpoint} `}
```

Comment:
Can this be simplified to:

```suggestion
      {label}: {s3Endpoint}
```